processors/crypto-processor/src/test/kotlin/net/corda/processors/crypto/internal/CryptoProcessorImplTest.kt

```diff
@@ -44,7 +50,7 @@ import org.mockito.kotlin.verify
 import org.mockito.kotlin.whenever
 import java.util.UUID
 
-class CryptoProcessorImplTest {
+class   CryptoProcessorImplTest {
```

Comment:
unnecessary new whitespace
```suggestion
class CryptoProcessorImplTest {
```